core/src/test/java/com/radixdlt/api/system/openapitools/ModelEqualsVerifierTest.java

```diff
@@ -1,7 +1,7 @@
 /*
  * Copyright 2021 Radix Publishing Ltd incorporated in Jersey (Channel Islands).
  * Licensed under the Radix License, Version 1.0 (the "License"); you may not use this
- * file except in compliance with the License. You may obtain a copy of the License at:
+ * folder except in compliance with the License. You may obtain a copy of the License at:
```

Comment:
Oops?

Reply:
huh I wonder why spotless didn't catch that